src/compas_timber/connections/solver.py

```diff
@@ -264,3 +272,134 @@ def _exceed_max_distance(pa, pb, max_distance, tol):
     @staticmethod
     def _is_near_end(t, length, max_distance, tol):
         return abs(t) * length < max_distance + tol or abs(1.0 - t) * length < max_distance + tol
+
+    def find_plate_plate_topology(self, plate_a, plate_b, tol=TOLERANCE):
```

Comment:
I assume this is the public interface method for the plate intersection topology finding, so I'd mark all the others as private. Also I'd suggest to move all these into a new sovler class that focuses on plates, since this one really only deals with beams.